moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -116,42 +130,48 @@ void RobotModelLoader::configure(const Options& opt)
         std::string prefix = rdf_loader_->getRobotDescription() + "_planning/joint_limits/" + jlim[j].joint_name + "/";
 
         double max_position;
-        if (nh.getParam(prefix + "max_position", max_position))
+        if (node_->has_parameter(prefix + "max_position"))
         {
+          max_position = node_->get_parameter(prefix + "max_position").get_value<double>();
```

Comment:
This can throw an exception, as `get_value<double>()` implicitly casts the parameters which could have the wrong type. So it probably makes sense to wrap all of this in a try/catch (probably a big one is enough) and print an error mentioning the failed parameter name. Also, it would be shorter to just use the template function `bool Node::get_parameter(param_name, result)`.